src/index.js

```diff
@@ -0,0 +1,90 @@
+const WS = require("websocket").w3cwebsocket;
+const EventEmitter = require("events");
+const { spawn } = require("child_process");
+const fs = require("fs");
+const path = require("path");
+const { v4: uuidv4 } = require("uuid");
+const constants = require("./constants.js")
+const {getBinaryName, normalize, getAuthInfo, arrayBufferToBase64, base64ToBytesArray} = require("./utils.js");
+
+class NeutralinoApp extends EventEmitter {
```

Comment:
Why do you need to extend `EventEmitter`? Can we just create a simple class without using `extend`?

Reply:
To enable app to emit events we need to extend the class with `EventEmitter`, I referenced it from [here](https://nodejs.org/en/learn/asynchronous-work/the-nodejs-event-emitter).

```js
app.on("ready", () => {
    console.log("App is ready")
})
```

Since we are not connecting with WS right now, I have removed the import for now, and will include in coming PR.

Sir if you have suggestion for `EventEmitter` replacement please let me know.